code/datums/status_effects/buffs/stun_absorption.dm

```diff
@@ -28,6 +28,11 @@
 	var/self_message
 	/// Message shown on anyone examining the owner.
 	var/examine_message
+	/// If TRUE, after passing the max seconds of stuns blocked, we will delete ourself.
+	/// If FALSE, we will instead recharge after some time.
+	var/delete_after_passing_max = TRUE
+	/// If [delete_after_passing_max] is FALSE, this is how long we will wait before recharging.
+	var/recharge_time = 1 MINUTES
```

Comment:
I would remove the defaults from here, in case confused future contributors try to change the defaults by modifying these only to have them overwritten by the defaults we _actually_ use in the `on_creation` args.